apps/easypid/src/crypto/bPrime.ts

```diff
@@ -407,6 +408,7 @@ export const requestSdJwtVcFromSeedCredential = async ({
         }
       },
       ...tokenResponse,
+      // @ts-ignore
```

Comment:
can't this be removed?

Reply:
Yeah so you'd prefer removing all bprime code?

Reply:
Ah no this is fine, no point actually in removing it :)